DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyBase.cs

```diff
@@ -393,6 +412,21 @@ public override void Initialize()
             AddChild(btnPickRandomMap);
         }
 
+        private void BtnGameModeMapSortAlpha_LeftClick(object sender, EventArgs e)
+        {
+            UserINISettings.Instance.GameModeMapSortState.Value = (int)btnGameModeMapSortAlpha.GetState();
+
+            RefreshGameSortAlphaBtn();
+            UserINISettings.Instance.SaveSettings();
+            ListMaps();
+        }
+
+        private void RefreshGameSortAlphaBtn()
```

Comment:
Name is misleadingly shortened, maybe `RefreshMapSortButton` would be better? Also we don't usually use control name shortenings in method names I think.

Reply:
Updated